doc/tutorials/vcat_configuration.rst

```diff
@@ -0,0 +1,87 @@
+.. _vcat_configuration:
+
+Enable vCAT Configuration
+#########################
+
+vCAT is built on top of RDT, so to use vCAT we must first enable RDT.
+For details on enabling RDT configuration on ACRN, see :ref:`rdt_configuration`.
+For details on ACRN vCAT high-level design, see :ref:`hv_vcat`.
+
+The vCAT feature is disabled by default in ACRN. You can enable it with these settings:
+
+#. Configure system level features:
```

Comment:
I'm assuming we can do this via the UI. If correct, we should add a note telling the user this is done via the UI. We can keep the XML section below indicating that it's listed here as an FYI to show them how those settings are translated into XML in the scenario file.

Reply:
yes, we should be able to do this via UI, but current UI or 2.8 may not support vcat config until  2.9, etc.